ui/partials/perf/graphsctrl.html

```diff
@@ -16,8 +16,8 @@
                 <td width="10px" class="graph-legend" style="padding: 4px; background-color: {{series.blockColor}}" ></td>
                 <td class="graph-legend" ng-class="{'series-inactive':!series.active}">
                   <b>{{series.name}}</b><br/>
-                  {{series.projectName}}<br/>
-                  {{series.platform}}<br/>
+                  <a ng-click="addRelatedBranches(series.signature)" tooltip="click here to add related branch.">{{series.projectName}}</a><br/>
+                  <a ng-click="addRelatedPlatform(series.signature)" tooltip="click here to add related platforms.">{{series.platform}}</a><br/>
```

Comment:
A few things with these:

1. Add a `href="#"` so that the cursor changes when you hover over them
2. Let's use `title` instead of `tooltip` for these, since tooltip is harder to read and we're using that elsewhere for additional information, rather than help.
3. For text, omit the "click here" terminology. Generally it's not great practice (e.g. http://www.cs.tut.fi/~jkorpela/www/click.html). Also omit the period and capitalize. So: "Add related branches" and "Add related platforms"